DQMOffline/L1Trigger/python/L1TTauDiff_cfi.py

```diff
@@ -45,11 +45,11 @@
 l1tTauEmuDiff = l1tDiffHarvesting.clone(
     plotCfgs=cms.untracked.VPSet(
         cms.untracked.PSet(  # EMU comparison
```

Comment:
Shouldn't this be converted to a dict and drop type afterward?

Reply:
Yeah, I have tried that, but it was showing error. That's why I left them unchanged. May I know if Pset->dist, then VPSet->??